envoy-control-tests/src/main/resources/lua_spec/ingress_rbac_logging_spec.lua

```diff
@@ -352,7 +361,18 @@ describe("envoy_on_response:", function()
             envoy_on_response(handle)
 
             -- then
-            assert.spy(handle.logInfo).was_called_with(_, formatLog("POST", "/path?query=val", "127.1.1.3", "service-first", "https", "", "403", false, false))
+            assert.spy(handle.logInfo).was_called_with(_, formatLog(
+                "POST",
+                "/path?query=val",
+                "127.1.1.3",
+                "service-first",
+                "https",
+                "",
+                "403",
+                false,
+                false,
+                "denied"
```

Comment:
Changes in this file are mostly formatting but this last field of "rbac_accion" was added. Pay attention to this field.